homeassistant/components/gogogate2/manifest.json

```
@@ -1,8 +1,8 @@
 {
   "domain": "gogogate2",
-  "name": "Gogogate2",
+  "name": "Gogogate2 and iSmartGate",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/gogogate2",
-  "requirements": ["gogogate2-api==1.0.4"],
+  "requirements": ["gogogate2-api==2.0.0"],
```

```suggestion
  "requirements": ["gogogate2-api==2.0.0"],
  "homekit": {
    "models": ["iSmartGate"]
  },  
```

These are discoverable via `homekit`.  `tradfri` has an example on how to implement that in `config_flow.py`

---

I cannot get HASS to detect this device.
My configuration.yml
```yaml
frontend:

api:

http:

config:

default_config:
discovery:
homekit:
```

```python
class Gogogate2FlowHandler(ConfigFlow, domain=DOMAIN):
    """Gogogate2 config flow."""

    VERSION = 1
    CONNECTION_CLASS = config_entries.CONN_CLASS_LOCAL_POLL

...

  async def async_step_homekit(self, discovery_info):
        """Handle homekit discovery."""
        print("DDDDDDDDDD", discovery_info)
        return await self.async_step_user()
...
```

The print statement never prints. I fired up wireshark and don't see any packets originating from the iSmartGate device until I visit the web interface. Any ideas?

---

I think HomeKit has to be turned on from the device app.

`python -m netdisco dump`

...
```
homekit:
 {'host': '192.168.209.201',
  'hostname': 'iSmartGate.local.',
  'name': 'iSmartGate',
  'port': 4567,
  'properties': {'c#': '5',
                 'ci': '2',
                 'ff': '1',
                 'id': '30:99:E3:35:58:9E',
                 'md': 'iSmartGate',
                 'pv': '1.1',
                 's#': '1',
                 'sf': '0',
                 'sh': 'JWSuyw=='}},
```
...

---

@vangorra If you can't get the it to show up, I'm happy to add the homekit discovery in a followup PR once this one it merged.

---

Good call. I'm getting zero devices showing up.
```
$ python -m netdisco dump
Discovered devices:
Discovered 0 devices
```